chapters/system_architecture/system_architecture.tex

```diff
@@ -1,4 +1,5 @@
 \chapter{System Architecture} \label{ch:system_architecture}
 This section describes the system architecture of the game.
 \input{chapters/system_architecture/sections/terrain/terrain.tex}
```

Comment:
We might want to have the chunk worker in terrain section. If not it should for sure be right after it